style.css

```diff
@@ -80,3 +80,515 @@ body {
   gap: 20px;
   padding: 0;
 }
+
+.card-works {
+  border: 1 px solid #fff;
+  margin-top: 142px;
+  position: absolute;
+  width: 100%;
+  top: 950px;
+  display: contents;
+  justify-items: center;
+}
+
+.card {
+  background: #fff;
+  border: 10px solid #fff;
+  border-radius: 24px;
+  margin: auto;
+  width: auto;
+  height: auto;
+}
```

Comment:
- [ ] The content of each project should be aligned using flex. So please make sure you use flex for the layout of each project.